app/assets/javascripts/application.js

```diff
@@ -100,7 +100,7 @@ function onContentLoad(){
         $(this).html(response);
       }
```

Comment:
It's not efficient to repeat jquery operations. For example, in this function, $(this) should be cached at the top of the function, like this: 
```js
var $this = $(this);
```
and then $(this) replaced with $this throughout the function like this:
```js
$(this).removeAttr('data-ajax-url');
```
becomes
```js
$this.removeAttr('data-ajax-url');
```